src/catalog/src/kvbackend/manager.rs

```diff
@@ -303,36 +301,70 @@ impl CatalogManager for KvBackendCatalogManager {
             }
         });
 
-        let table_id_stream = self
-            .table_metadata_manager
-            .table_name_manager()
-            .tables(catalog, schema)
-            .map_ok(|(_, v)| v.table_id());
         const BATCH_SIZE: usize = 128;
-        let user_tables = try_stream!({
+        const CONCURRENCY: usize = 8;
+
+        let (tx, rx) = tokio::sync::mpsc::unbounded_channel();
```

Comment:
I think the bounded channel should also work well

Reply:
> I think the bounded channel should also work well

We don't explicitly set the bound here since the concurrency if stream if harnessed.

Reply:
> I think the bounded channel should also work well

fixed in 1d55a5f3cdd0e3d6bea62cf5948a91bafb8e7c65